### ISSUE TITLE
Non Transferable and Delegation

### ISSUE THREAD

**Issue by Suelain** [ChristopherA/self-sovereign-identity#2]
Hi Christopher,

The principles could be enhanced by stating as well that a person's identity can't be transferable to someone else.  It shouldn't be possible for a person to willingly or be forced to give their own identity to a different person.  

Another group to consider is children or others needing help to establish and maintain their identity until they can do this themselves.  This might be more of a requirement than a principle, but for those who can't create their own identity, someone should be delegated to do so and act on their behalf.

**Comment by ChristopherA** [comment]
Makes sense — I will discuss these with others, and see if we can't integrate it in the next version. Thanks!
